source/schemas/types.txt

```diff
@@ -509,3 +509,78 @@ with the ``title`` field set to the embedded object's type name:
             }
       }
    }
+
+.. _geospatial
+
+
+Geospatial Data
+---------------
+
+Geospatial data describes points and other data on the earth's surface. App 
+Services supports defining GeoJSON Point data in a schema. 
+
+GeoJSON Point
+~~~~~~~~~~~~~
+
+A GeoJSON Point is a single location on the Earth's surface, described by its 
+longitude and latitude. To ensure wide compatibility, App Services stores these 
+points in accordance to the `GeoJSON spec <https://datatracker.ietf.org/doc/html/rfc7946>`_, 
+which has additional requirements. Specifically, you must apply the following 
+rules to the GeoPoint:
+
+- It must be of type ``object``
+- It must be embedded within another type
+- It must contain the following fields:
+
+  - ``type``, of type ``string``, which is always set to "Point"
+  - ``coordinates``, of type ``array``. The ``coordinates`` array contains 
+    exactly 2 double values. The first double is the longitude of the point and 
+    the second is the latitude.
+
+- Additional properties are allowed.
+ 
+ The following is the schema of a GeoPoint:
+
+.. code-block:: json
+
+   {
+     "title": "MyObject",
+     "properties": {
+       "location": {
+         "bsonType": "object",
+         "required": [ "type" ],
+         "properties": {
+           "type": {
+             "bsonType": "string",
+           },
+           "coordinates": {
+             "bsonType": "array",
+             "items": {
+               "bsonType": "double"
+             }
+           }
+         }
+       },
+       "another_property": {
+             "bsonType": "double"
+       },
+     }
+   }
+
+Querying GeoJSON Points
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can query geospatial points when using :ref:`Flexible Sync <flexible-sync>`. 
```

Comment:
Leads me to wonder how I do geo queries with GraphQL & Data API (and maybe even MQL queries in Functions). I assume the current GeoSpatial project is only scoped for Sync and not the other services?

Reply:
Correct - FlexSync only